recipes/uwsgi/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: uwsgi
+  version: 2.0.12
+
+source:
+  fn: "uwsgi-2.0.12.tar.gz"
+  url: "https://pypi.python.org/packages/0c/1c/44849e293e367a157f1ad863cee02b4b865840543254d8fae3ecdebdbdb9/uwsgi-2.0.12.tar.gz"
```

Comment:
We also like to template the version info using jinja. It keeps things simple when updating. See this as an [example]( https://github.com/conda-forge/envisage-feedstock/blob/32c9b5a9b4bc2d2f37add6d52bc93cf7b7759f0d/recipe/meta.yaml ).